package/gluon-core/luasrc/lib/gluon/upgrade/115-swconfig

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/lua
+
+local platform = require 'gluon.platform'
+local uci = require('simple-uci').cursor()
+
+local function get_keys(t)
+	local keys={}
+	local n=0
+
+	for k,_ in pairs(t) do
+		n=n+1
+		keys[n]=k
+	end
+	return keys
+end
+
+local switch_vlans = {
+	-- device identifier, lan ports, wan ports
+	["ubnt,unifiac-pro"]={"2 0t", "3 0t"},
+}
+
+local devices = get_keys(switch_vlans)
+
+
+if platform.match('ath79', 'generic', devices) then
```

Comment:
How about `if not platform.match ... then return fi`

Reply:
resolved in 7cab846e9ceed11755c29030864962bc345da2ef